smartsim/_core/mli/message_handler.py

```diff
@@ -92,7 +92,9 @@ def build_output_tensor_descriptor(
         return description
 
     @staticmethod
-    def build_tensor_key(key: str) -> data_references_capnp.TensorKey:
+    def build_tensor_key(
+        key: str, feature_store_descriptor: str
+    ) -> data_references_capnp.TensorKey:
```

Comment:
Need to update doc string for ``feature_store_descriptor``

Reply:
Fixed

Reply:
Fixed